src/averageAges.js

```diff
@@ -36,8 +46,19 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
-function calculateWomenAverageAge(people, withChildren) {
+function calculateWomenAverageAge(people, withChildren = false) {
   // write code here
+  const WOOMEN = people.filter(person => person.sex === 'f');
+  const FILTERED_WOOMEN = WOOMEN.filter(woman =>
+    withChildren ? people.some(person => person.mother === woman.name) : true
```

Comment:
If `withChildren` is false we don't need this filtering. Try to change this logic(as you did in first function for example)